components/Layout/FooterSlackCTA/FooterSlackCTA.tsx

```diff
@@ -0,0 +1,19 @@
+import styles from './FooterSlackCTA.module.scss';
+
+export const FooterSlackCTA = () => (
+    <div className={styles.container}>
+        <div className={styles.emoji}>👂</div>
+        <div>
+            <div className={styles.cta__text}>Hear the latest news…</div>
+            <div className={styles.cta}>
+                Join{' '}
+                <a href="https://github.com/wagtail/wagtail/wiki/Slack">
+                    <strong className={styles.cta__tag}>#headless</strong>
+                </a>{' '}
+                on Slack
```

Comment:
There's something up with the preview site Vercel deployed for this branch not loading the current code correctly - for this footer cta, the entire text is a link, not just the #headless.

<img width="675" alt="image" src="https://user-images.githubusercontent.com/18164832/155692806-81f04bf8-1fa2-4b9d-b39e-a8530ca563fc.png">

Reply:
The preview deployment for this branch has been taken down I think, Vercel is responding with a 404 when I try to view it. Maybe it defaults to an older deployment?